client/css/components/results.scss

```diff
@@ -1,14 +1,30 @@
 
 
 // Component
-.ql-results-page, .ql-grades-overview {
+.ql-grade-page, .ql-grades-overview {
+
+  overflow-y: hidden; // hide vertical
+  overflow-x: hidden; // hide horizontal
 
   .session-group {
-    margin-top: 20px;
+    padding-top: 20px;
+  }
+
+  .session-view{
+    padding-right: 5px;
+  }
+.ql-button-centered{
+  width: 100%;
+  margin-left: 43%;
+  margin-right: 40%;
+  padding: 7px;
+}
+  .ql-shift-button{
+    margin-left: 850px;
```

Comment:
This isn't good practice to anchor something to the right hand part of the screen.